www/src/pages/foundations/typography.tsx

```diff
@@ -42,356 +101,233 @@ const measuredTypeProps = {
-              </td>
-            </tr>
+                ),
+              },
+              {
+                Header: 'Mobile',
+                Cell: ({ row }: DataTableTypes) => (
+                  <div className="mobile-type">
+                    <MeasuredItem {...measuredTypeProps}>
+                      <p className={`m-0 display-${row.original.size}`}>
+                        Display {row.original.size}
+                      </p>
+                    </MeasuredItem>
+                  </div>
+                ),
+              },
+              {
+                Header: 'CSS Class',
+                Cell: ({ row }: DataTableTypes) => (
+                  <code style={{ fontSize: '16px' }}>
+                    .display-{row.original.size}
+                  </code>
+                ),
+              },
+            ]}
+          >
+            <DataTable.Table />
+          </DataTable>
+        </div>
 
-            <tr>
-              <td>
-                <p className="text-italic">Italic text.</p>
-              </td>
-              <td>
-                <code>.text-italic</code>
-              </td>
-            </tr>
+        <h2 className="mb-2">Links</h2>
+        <div className="mb-4">
+          <DataTable
+            itemCount={3}
+            data={linksClassesAndDescriptions.map(({ name, text }) => ({ name, text }))}
+            columns={[
+              {
+                Header: 'Desktop',
+                Cell: ({ row }: DataTableTypes) => (
+                  <p
+                    className={`m-0 ${row.original.name}`}
+                    dangerouslySetInnerHTML={{ __html: row.original.name }}
+                    style={{ fontSize: '16px' }}
+                  />
+                ),
+              },
+              {
+                Header: 'Mobile',
+                Cell: ({ row }: DataTableTypes) => (
+                  <p
+                    className={`m-0 ${row.original.text}`}
+                    dangerouslySetInnerHTML={{ __html: row.original.text }}
+                    style={{ fontSize: '16px' }}
+                  />
+                ),
+              },
+            ]}
+          >
+            <DataTable.Table />
+          </DataTable>
+        </div>
 
-            <tr>
-              <td>
-                <p className="font-weight-bold">Bold text.</p>
-              </td>
-              <td>
-                <code>.font-weight-bold</code>
-              </td>
-            </tr>
-            <tr>
-              <td>
-                <p className="font-weight-normal">Regular text.</p>
-              </td>
-              <td>
-                <code>.font-weight-normal</code>
-              </td>
-            </tr>
-          </tbody>
-        </table>
+        <h2 className="mb-2">Decoration and Emphasis</h2>
+        <div className="mb-4">
+          <DataTable
+            itemCount={2}
+            data={decorationAndEmphasisClassesAndDescriptions.map(({
```

Comment:
no need to do `.map` here, `data={decorationAndEmphasisClassesAndDescriptions}` should work

Reply:
Thanks, simplified